unlock-app/src/components/creator/CreatorLockForm.js

```diff
@@ -22,7 +22,7 @@ class CreatorLockForm extends React.Component {
       keyPrice: '0.01',
       keyPriceCurrency: 'ether',
       maxNumberOfKeys: 10,
-      name: 'New Lock',
+      lockName: 'New Lock',
```

Comment:
Could we keep name here? Since we're already in the `LockForm` I feel like calling it lockName is redundant. Also for consistency since no other attribute starts with `lock`...

Reply:
OK!